include/wsrep/server_state.hpp

```diff
@@ -355,6 +355,10 @@ namespace wsrep
          *  Return current view
          */
         const wsrep::view& current_view() const { return current_view_; }
+        /**
+         *  Return the previous primary view
+         */
+        const wsrep::view& previous_primary_view() const { return previous_primary_view_; }
```

Comment:
I think the `previous_primary_view_` should be private for `server_state`, therefore I'd leave this getter methoud out of the public interface.

Reply:
you're right, getter removed